benchmarks/pool_stress_test.cpp

```diff
@@ -0,0 +1,93 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include <iostream>
+#include <chrono>
+#include <string>
+#include <random>
+#include <numeric>
+
+#include "umpire/config.hpp"
+#include "umpire/ResourceManager.hpp"
+#include "umpire/Allocator.hpp"
+#include "umpire/strategy/MixedPool.hpp"
+
+#define CONVERT 1000000 //convert sec (s) to microsec (us)
+
+//Set up test factors
+const long long unsigned int ALLOC_SIZE {137438953472}; //137GiB, total size of all allocations together
+const int SIZE {1<<28}; //268MiB, size of each allocation
+const int NUM_ALLOC {512}; //number of allocations for each round
+const int NUM_RND {1000}; //number of rounds (used to average timing)
+
+void run_test(umpire::Allocator alloc, std::vector<int> &indices, std::string test_name, std::string pool_name)
+{
+  double time[2] = {0.0, 0.0};
+  void* allocations[NUM_ALLOC];
```

Comment:
I think it would be best to use a vector here and limit the use of constants to just a few places in the code.  For this function, the number of allocations to be performed is directly tied to the number of indices being passed in.  I suggest that this should be changed to:
```suggestion
  std::vector<void*> allocations(indices.size());
```

Reply:
Done